dev/tools/merge-pr.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh -e
+
+# This script depends (at least) on git and jq.
+# It should be used like this: dev/tools/merge-pr.sh /PR number/
+
+#TODO: check arguments and show usage if relevant
+
+PR=$1
+
+git fetch github
+
+BASE_BRANCH=`curl https://api.github.com/repos/coq/coq/pulls/$PR 2>/dev/null | jq -r '.base.label'`
+LOCAL_BASE_BRANCH=`git rev-parse --abbrev-ref HEAD`
+
+COMMIT=`git rev-parse github/pr/$PR`
+STATUS=`curl https://api.github.com/repos/coq/coq/commits/$COMMIT/status 2>/dev/null | jq -r '.state'`
```

Comment:
Looks like `API=https://api.github.com/repos/coq/coq` would make the script a bit more readable, as well as calling `curl -s`